satellite/console/consoleweb/consoleql/mutation.go

```diff
@@ -83,60 +79,26 @@ func rootMutation(log *zap.Logger, service *console.Service, mailService *mailse
 					input, _ := p.Args[InputArg].(map[string]interface{})
 					secretInput, _ := p.Args[Secret].(string)
 
-					offerType := rewards.FreeCredit
-
 					createUser := fromMapCreateUser(input)
 
 					secret, err := console.RegistrationSecretFromBase64(secretInput)
 					if err != nil {
-						log.Error("register: failed to parse secret",
-							zap.String("rawSecret", secretInput),
+						log.Error("register: failed to create account",
```

Comment:
@ifraixedes Do you think this is a good way to do it?

Reply:
I think so according our conventions.

I would add to the `Debug` message something like `"register: failed to create an account with: "

It will be easier to related the this error message  with the debug one